compiler/fir/analysis-tests/testData/extendedCheckers/RedundantModalityModifierChecker.kt

```diff
@@ -1,3 +1,27 @@
+// Interface
+interface Interface {
+    // Redundant
+    <!REDUNDANT_MODALITY_MODIFIER!>open<!> val gav: Int
```

Comment:
Please leave here a comment that `REDUNDANT_MODALITY_MODIFIER` is questionable here (in FE 1.0, compiler reports a warning here, and inspection doesn't report anything).